### PR TITLE
Added Loops

### PR DESCRIPTION
**Pull request by todorzag**

### CODE REVIEW

File: Loops/Problem10/Program.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int n = int.Parse(Console.ReadLine());
+
+            int[,] matrix = new int[n, n];
+
+            int counter = 1;
```
Comment (mvlt) [inline Loops/Problem10/Program.cs]: Never used

File: Loops/Problem10/Program.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+            for (int row = 0; row < n; row++)
```
Comment (mvlt) [inline Loops/Problem10/Program.cs:11]: Why need to create and populate matrix and have 4 cycles couldn't be done with less cycles?

File: Loops/Problem11/Program.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            int result = GetNumberOfZerosAtEnd(factoriel);
+            Console.WriteLine(result);
+        }
+
+        public static int GetFactoriel(int n)
+        {
+            int fact = 1;
+            while (true)
+            {
+                if (n == 1)
```
Comment (mvlt) [inline Loops/Problem11/Program.cs]: Could we skip this if statement evaluation in each iteration?

File: Loops/Problem11/Program.cs [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+            return fact;
+        }
+
+        public static int GetNumberOfZerosAtEnd(int factoriel)
+        {
+            string factorielString = factoriel.ToString();
+            int counter = 0;
+
+            for (int x = factorielString.Length; x > 0; x--)
```
Comment (mvlt) [inline Loops/Problem11/Program.cs]: Do we need this hole logic for just counting end 0s?

File: Loops/Problem12/Program.cs [hunk 5]
```diff
@@ -0,0 +1,30 @@
+﻿namespace Problem12
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int quotient = int.Parse(Console.ReadLine());
+
+            Stack<int> stack = new Stack<int>();
```
Comment (mvlt) [inline Loops/Problem12/Program.cs:9]: Why do we need to use
Stack<int> stack = new Stack<int>(); String?

File: Loops/Problem13/Program.cs [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    {
+        static void Main(string[] args)
+        {
+            string binaryNum = Console.ReadLine();
+
+            double result = 0;
+
+            for (int i = 0; i < binaryNum.Length; i++)
+            {
+                int toThePowerOf = binaryNum.Length - i - 1;
```
Comment (mvlt) [inline Loops/Problem13/Program.cs:22]: Leave comments to explain the behavior. What I will expect if %10 and /10 like in previous example or  binaryNum[i]) * Math.Pow(2, i)

File: Loops/Problem14/Program.cs [hunk 7]
```diff
@@ -0,0 +1,52 @@
+﻿namespace Problem14
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int quotient = int.Parse(Console.ReadLine());
+
+            Stack<string> result = new Stack<string>();
```
Comment (mvlt) [inline Loops/Problem14/Program.cs:9]: Why do we need
Stack<string> result = new Stack<string>();

File: Loops/Problem14/Program.cs [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+            }
+
+            PrintResult(result);
+        }
+
+        public static string CheckRemainder(int remainder)
+        {
+            string result = remainder.ToString();
+
+            Dictionary<int, string> moreThan10 = new Dictionary<int, string>
```
Comment (mvlt) [inline Loops/Problem14/Program.cs]: CheckReminder() do we need to create every time moreThan10 Dictionary and do we need to do it for every remainder?

File: Loops/Problem14/Program.cs [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+            int quotient = int.Parse(Console.ReadLine());
+
+            Stack<string> result = new Stack<string>();
+
+            while (quotient != 0)
+            {
+                int remainder = quotient % 16;
+                quotient /= 16;
+
+                result.Push(CheckRemainder(remainder));
```
Comment (mvlt) [inline Loops/Problem14/Program.cs]: Can we get the right characters without using CheckReminder() at all? Try Convert.ToChar()

File: Loops/Problem16/Program.cs [hunk 10]
```diff
@@ -0,0 +1,43 @@
+﻿namespace Problem16
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int n = int.Parse(Console.ReadLine());
+            int[] array = new int[n];
+
+            Random random = new Random();
```
Comment (mvlt) [inline Loops/Problem16/Program.cs:10]: Never used

File: Loops/Problem16/Program.cs [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+
+            for (int i = 1; i <= n; i++)
+            {
+                int index = GetRandomIndex(array);
```
Comment (mvlt) [inline Loops/Problem16/Program.cs]: Do we need a recursion? Try swapping method after populating the array

File: Loops/Problem12/Program.cs (hunk 5 above)
Comment (todorzag) [reply Loops/Problem12/Program.cs:9]: Because the formula for conversion must have the numbers in opposite order

**Pull request merged**